web/php/Modules/Forum/ForumViewThreadModule.php

```diff
@@ -168,6 +168,8 @@ public function build($runData)
         $runData->contextAdd("thread", $thread);
         $runData->contextAdd("category", $category);
         $runData->contextAdd("posts", $posts);
+        
+        $runData->contextAdd("canDelete", WDPermissionManager::instance()->hasForumPermission('moderate_forum', $runData->getUser(), $category));
```

Comment:
As mentioned in chat, I think this is more readable:
```suggestion
        $canDelete = WDPermissionManager::instance()->hasForumPermission('moderate_forum', $runData->getUser(), $category);
        $runData->contextAdd("canDelete", $canDelete);
```